### DISCUSSION TITLE
Persistent indent style configuration

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#1913] [Q&A]
From this: https://github.com/helix-editor/helix/discussions/1291#discussioncomment-1841370, I know that I can set the tab width.

I would like to set that not to a specific language nor manually when with a open buffer, instead, I would like to have a key for this in `config.toml` just like we have for `languages.toml`, but for all languages/buffers. Is it possible? Because I couldn't find anything documented in [Configuration section](https://docs.helix-editor.com/configuration.html).

**Comment by whoizit** [comment]
I need to change the default indent, for files with no extension or type, just for text files

UPD: 
this works for files without ext:
```
[[language]]
name = "any"
scope = ""
file-types = []
roots = []
indent = { tab-width = 2, unit = "  " }
```

UPD:
not works if I run `hx` without filename
